stage-0-getting-started.md

```diff
@@ -48,5 +48,12 @@ Here are some starting points for learning about the Ethereum protocol.
    - Stateless Ethereum: https://blog.ethereum.org/2019/12/30/eth1x-files-state-of-stateless-ethereum/
    - Kate Commitments: https://hackmd.io/yqfI6OPlRZizv9yPaD-8IQ?view#Background
 
+#### Sharding
+   - Overview: https://ethereum.org/en/eth2/shard-chains/
+   - FAQ: https://eth.wiki/sharding/Sharding-FAQs
+   - Research Compendium: https://notes.ethereum.org/@serenity/H1PGqDhpm?type=view#External-links
```

Comment:
I think that it's very possible these links are out of date and I don't want to publish them as relevant without being sure they are up-to-date. Here's one that is definitely up-to-date: https://vitalik.ca/general/2021/04/07/sharding.html
@pipermerriam 

I tend to be skeptical of anything labeled "Serenity" as those things are sometimes vestigial leftovers from many years ago.

Reply:
Good catch, and thanks for the info on "Serenity". I've added the Vitalik blog.

The FAQ builds on Vitalik's sharding blog post with quite a bit more detail, but it was last edited about a year ago. I'll put some time into determining whether it's up-to-date and re-include it in another pull request if I find that it is.